modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -137,6 +137,24 @@ def generic_schemas(ref_root)
           { '$ref': "#{ref_root}/date" }
         ]
       },
+      'X-VA-Claimant-First-Name': {
```

Comment:
Should all of the claimant headers be down here? Birthdate and SSN included?

Reply:
Yeah, I think we should. The list of schemas doesn't mean it's used everywhere, just that it's one of many available across our endpoints.